lib/lita-reviewme.rb

```diff
@@ -4,4 +4,14 @@
   File.join("..", "..", "locales", "*.yml"), __FILE__
 )]
 
+require "lita-reviewme/github"
+
 require "lita/handlers/reviewme"
+
+module Lita
+  module Reviewme
+    class NoReviewer < StandardError; end
+    class UnknownOwner < StandardError; end
+    class CannotPostComment < StandardError; end
```

Comment:
Thoughts on moving these GH-specific errors into the Github class? e.g. `Lita::Reviewme::Github::UnknownOwner`.